torchvision/models/quantization/googlenet.py

```diff
@@ -0,0 +1,174 @@
+import warnings
+from collections import namedtuple
+import torch
+import torch.nn as nn
+from torchvision.models.utils import load_state_dict_from_url
+import torchvision.models.googlenet
+import sys
+from torch import Tensor
+from torch.jit.annotations import Optional
+
+googlenet_module = sys.modules['torchvision.models.googlenet']
+
+__all__ = ['QuantizableGoogLeNet', 'googlenet', "QuantizableGoogLeNetOutputs", "_QuantizableGoogLeNetOutputs"]
+
+model_urls = {
+    # GoogLeNet ported from TensorFlow
+    'googlenet': 'https://download.pytorch.org/models/googlenet-1378be20.pth',
+}
+
+QuantizableGoogLeNetOutputs = namedtuple('QuantizableGoogLeNetOutputs', ['logits', 'aux_logits2', 'aux_logits1'])
+QuantizableGoogLeNetOutputs.__annotations__ = {'logits': Tensor, 'aux_logits2': Optional[Tensor],
+                                    'aux_logits1': Optional[Tensor]}
+
+# Script annotations failed with _QuantizableGoogleNetOutputs = namedtuple ...
+# _QuantizableGoogLeNetOutputs set here for backwards compat
+_QuantizableGoogLeNetOutputs = QuantizableGoogLeNetOutputs
+
+
+def googlenet(pretrained_float_model=False, progress=True, **kwargs):
+    r"""GoogLeNet (Inception v1) model architecture from
+    `"Going Deeper with Convolutions" <http://arxiv.org/abs/1409.4842>`_.
+
+    Args:
+        pretrained_float_model (bool): If True, returns a model pre-trained on ImageNet
+        progress (bool): If True, displays a progress bar of the download to stderr
+        aux_logits (bool): If True, adds two auxiliary branches that can improve training.
+            Default: *False* when pretrained_float_model is True otherwise *True*
+        transform_input (bool): If True, preprocesses the input according to the method with which it
+            was trained on ImageNet. Default: *False*
+    """
+    if pretrained_float_model:
```

Comment:
It looks like pretrained_float_model indentation is off. We should return a randomly initialized model with pretrained_float_model is False.

Reply:
Could you give more detail on which indentation is off? If pretrained_float_model is False line 61 will return QuantizableGoogLeNet(**kwargs), which is a randomly initialized model.